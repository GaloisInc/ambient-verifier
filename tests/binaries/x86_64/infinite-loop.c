/*
 * A test for which simulation should not terminate unless an iteration bound
 * is imposed.
 */

int main() {
  while (1) {}
  return 0;
}
