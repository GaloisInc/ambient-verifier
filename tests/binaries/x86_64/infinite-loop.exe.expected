successful: 0
failed: 1
iterationBound: 10
